akka-stream/src/main/scala/akka/stream/impl/ActorMaterializerImpl.scala

```diff
@@ -99,7 +100,7 @@ private[akka] case class ActorMaterializerImpl(
             assignPort(stage.outPort, processor)
             mat
 
-          case tls: TlsModule ⇒
+          case tls: TlsModule ⇒ // TODO isn't this a bit of a hack?
```

Comment:
There is no abstraction right now that allows to build n-to-n stages at the user level, unlike Stages for 1-to-1 or FlexiMerge/Route to 1-to-n or n-to-1 (and FlexiMerge/Route are limited). I need to actually implement the proper generalization of PushPullStage to arbitrary graph junctions. That is my next plan, since it was always postponed for various reasons.